cypress/integration/pages/articles/testsForCanonicalOnly.js

```diff
@@ -43,39 +43,61 @@ export const testsThatFollowSmokeTestConfigForCanonicalOnly = ({
       );
     });
 
-    // Have the test toggle between if mediaPlayer is enabled on live or not
-    let imageIndex = 1;
-    if (appToggles.mediaPlayer.enabled) {
-      imageIndex = 2;
-    }
-
     if (serviceHasCaption(service)) {
-      it('should have a visible image without a caption that is lazyloaded and has a noscript fallback image', () => {
-        cy.get('figure')
-          .eq(imageIndex)
-          .within(() => {
-            cy.get('div div div div').should(
-              'have.class',
-              'lazyload-placeholder',
+      describe('Image with placeholder', () => {
+        it('should have a visible image that is not lazyloaded', () => {
+          cy.get('div[class^="ImagePlaceholder"]')
+            .eq(0)
+            .within(() => {
+              cy.get('div[class^="lazyload-placeholder"]').should('not.exist');
```

Comment:
This describe block says "Image with placeholder" but here we're asserting the existence of an image _without_ a placeholder, this seems confusing. Can we think again about the naming of these tests? What are we actually checking?

Reply:
Generally the component that renders the image is `<ImageWithPlaceholder />`, so basically we are testing for its attributes whether its has a caption or not, or whether it is lazyloaded.

Reply:
Can we still test that the image exists, as we are explicit in stating we are asserting there is a `visible image`?

Reply:
yeah, just added the assertion